cmd/manager/main.go

```diff
@@ -67,7 +72,7 @@ var (
 	defaultSystemNamespace = "olmv1-system"
 )
 
-const authFilePath = "/etc/operator-controller/auth.json"
+const authFilePrefix = "operator-controller-global-pull-secrets.json"
```

Comment:
`authFilePrefix` might be a misleading name since this looks more like a file name which I would expect to be considered a "suffix" on the path.

Reply:
Actually the `.json` should not have been there, the fmt.Sprintf directive down below already includes the `.json`. 

Should be fixed now, good catch! 

here's what it looks like now: https://go.dev/play/p/x35HITorSPj